views/index.erb

```diff
@@ -12,4 +12,11 @@
     <input type='password' name='password_confirmation' placeholder='Confirm Password'>
     <input type='submit' value='Sign up'>
   </form>
+
+  <% if flash[:nomatch] %>
+    <div id='flash' class='nomatch'>
+      <a class="close" data-dismiss="alert"><%= flash[:nomatch] %></a>
+    </div>
+  <% end %>
+  
```

Comment:
I'm sure we can simply use:   <%= flash[:nomatch] %> in the view.

Reply:
That's good to know, I've updated the relevant parts of code, and confirmed that it still passes rspec and performs as expected in rackup.